src/hooks/useTransactionType.tsx

```diff
@@ -131,7 +132,7 @@ export const getTransactionType = (tx: TransactionSummary, addressBook: AddressB
         }
       }
 
-      if (isOnChainConfirmationTxInfo(tx.txInfo)) {
+      if (isOnChainConfirmationTxInfo(tx.txInfo) || isExecTxInfo(tx.txInfo)) {
```

Comment:
As before re. the influence on existing `execTransaction` calls. Won't this try to decode "standard" transactions as well?

Reply:
Which others? Sadly we lack detailed data in the TxInfo type. Ideally we would also get the method selector which is less ambiguous as it also hashes the parameter types.

But I am not aware of other execTransaction transactions.

Reply:
As per discussion: we always display the transaction executed so the `execTransaction` here is of the nested Safe.